### PR TITLE
[ID] The people Category

### PR DESCRIPTION
**Pull request by vzrenggamani**
### Description

- The People and Staff update list
- Translations some of documents
- Rewrite some outdated docs

### Status

- [x] WIP
- [x] Pending Review
- [ ] Finished

### Additional Notes 
` I've deleted the oudated tag`

### CODE REVIEW

File: wiki/People/The_Team/id.md [hunk 1]
```diff
@@ -1,61 +1,74 @@
...
-| [alumni osu!](/wiki/People/osu!_Alumni) | Orang-orang yang dikenal atas kontribusi mereka setelah pindah |
-| [Tim Pendukung Zaman Dulu](/wiki/People/Support_Team) | Dukungan Teknis/Bantuan Permintaan Fitur |
-| [Berita osu!](http://osu.ppy.sh/g/25) | Menyediakan Anda dengan kualitas bacaan sejak 2013 |
-| [Karyawan Turnamen](http://osu.ppy.sh/g/26) | Responsible for handling officially supported osu! tournaments across all gamemodes. |
-| [Tim Penjamin Asuransi](/wiki/People/Quality_Assurance_Team) | Kami membantu Anda menikmati osu!, mulai dari menjamin kualitas beatmap. |
+| [Global Moderation Team](/wiki/People/Global_Moderation_Team) | Mengawasi percakapan pada forum dan obrolan dalam game |
+| [Beatmap Nomination Group](/wiki/People/Beatmap_Nomination_Group) | Pengguna yang datang untuk memastikan beatmap anda memenuhi syarat |
+| [Developers](https://osu.ppy.sh/g/11) | Membuat game lebih memukau dan mengatasi bug |
+| [osu! Alumni](/wiki/People/osu!_Alumni) | Mereka yang telah berkontribusi dan telah berhenti |
+| [Support Team Redux](/wiki/People/Support_Team) | Penangana Permintaan bantuan maupun saran |
```
Comment (Rendyindo) [inline wiki/People/The_Team/id.md]: Penangana -> Penanganan

File: wiki/People/The_Team/id.md [hunk 2]
```diff
@@ -1,61 +1,74 @@
...
+| [Quality Assurance Team](/wiki/People/Quality_Assurance_Team) | Kami membantu Anda menikmati osu !, dimulai dengan memastikan kualitas beatmap. |
```
Comment (Rendyindo) [inline wiki/People/The_Team/id.md]: Don't add space between "osu" and it's exclamation mark

**Comment by TPGPL** [comment]
resolve conflicts

**Comment by Joehuu** [comment]
Probably want to give(merge) these changes to #1493. And update it accordingly with the PR’s `en.md`.

**Comment by TPGPL** [comment]
Could you undo the pull request template changes?

**Comment by TPGPL** [comment]
closing as there's no activity nor response

**Pull request closed without merging**